recipes-core/systemd/systemd/imx-generic-bsp/host-watchdog-poweroff.service

```diff
@@ -0,0 +1,22 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is for host monitor x or related products.
+#  It require the host-watchdog-driver( and  firmware) to safely shutdown the unit.
+#
+#  The command execute shutdown on m7(cocpu) after 5 seconds and if the system has start signal set (high),
+#  the unit will instead of shutdown reboot. This behavior is the same for older Host mobility products.
+#  Shutdown(cocpu) will release the internal start signal and after a timeout time it will toggle reset pin.
+#  This service will work with system 'halt' or a systemd-poweroff.service with halt instead of shutdown.
+
+[Unit]
+Description=Host Watchdog Shutdown Service
+DefaultDependencies=no
+Before=shutdown.target
+
+[Service]
+Type=oneshot
+ExecStart=/bin/bash -c '/bin/echo 5 > /sys/class/host_watchdog/ctl_shutdown'
```

Comment:
```suggestion
ExecStart=/bin/sh -c '/bin/echo 5 > /sys/class/host_watchdog/ctl_shutdown'
```

Reply:
/bin/sh points to /bin/bash right now, but one might want to use a different default shell than bash at some point.